### ISSUE TITLE
Showing new listings for Friday, 1 November 2024

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#52]
## An occupation number quantum subspace expansion approach to compute the single-particle Green function: an opportunity for noise filtering
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** B. Gauthier, P. Rosenberg, A. Foley, M. Charlebois
 - **Subjects:** Subjects:
Strongly Correlated Electrons (cond-mat.str-el); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2312.13497
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2312.13497
 - **Abstract**
 We introduce a hybrid quantum-classical algorithm to compute the Green function for strongly correlated electrons on noisy intermediate-scale quantum (NISQ) devices. The technique consists in the construction of a non-orthogonal excitation basis composed of a set of single-particle excitations generated by occupation number operators. The excited sectors of the Hamiltonian in this basis can then be measured on the quantum device and a classical post-processing procedure yields the Green function in the Lehmann representation. The technique allows for noise filtering, a useful feature for NISQ devices. To validate the approach, we carry out a set of proof-of-principle calculations on the single-band Hubbard model on IBM quantum hardware. For a 2 site system we find good agreement between the results of quantum simulations and the exact result for the local spectral function. This proof-of-principle also shows that the noise filtering provides a reliable way to get rid of satellite peaks present in the spectral weight obtained from a NISQ device. A simulation of a 4 site system carried out on classical hardware suggests that the approach can achieve similar accuracy for larger systems.
## Quantum Digital Twins for Uncertainty Quantification
- **Keywords:** ['hybrid quantum']
 - **Authors:** Soronzonbold Otgonbaatar, Elise Jennings
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.23311
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.23311
 - **Abstract**
 Modern supercomputers can handle resource-intensive computational and data-driven problems in various industries and academic fields. These supercomputers are primarily made up of traditional classical resources comprising CPUs and GPUs. Integrating quantum processing units with supercomputers offers the potential to accelerate and manage computationally intensive subroutines currently handled by CPUs or GPUs. However, the presence of noise in quantum processing units limits their ability to provide a clear quantum advantage over conventional classical resources. Hence, we develop and construct "quantum digital twins," virtual versions of quantum processing units. To demonstrate the potential benefit of quantum digital twins, we create and deploy hybrid quantum ensembles on five quantum digital twins that emulate parallel quantum computers since hybrid quantum ensembles are suitable for distributed computing. Our study demonstrates that quantum digital twins assist in analyzing the actual quantum device noise on real-world use cases and emulate parallel quantum processing units for distributed computational tasks to obtain quantum advantage as early as possible.
## Deterministic and reconfigurable graph state generation with a single solid-state quantum emitter
- **Keywords:** ['semiconductor quantum dot']
 - **Authors:** H. Huet (1), P. R. Ramesh (1 and 2), S. C. Wein (3), N. Coste (1 and 4), P. Hilaire (3), N. Somaschi (3), M. Morassi (1), A. Lemaître (1), I. Sagnes (1), M. F. Doty (2), O. Krebs (1), L. Lanco (1 and 5), D. A. Fioretto (1 and 3), P. Senellart (1) ((1) Université Paris-Saclay, Centre de Nanosciences et de Nanotechnologies, CNRS, Palaiseau, France, (2) University of Delaware, Newark, DE, USA, (3) Quandela SAS, Massy, France, (4) School of Mathematical and Physical Sciences, University of Technology Sydney, Ultimo, New South Wales, Australia, (5) Université Paris Cité, Centre de Nanosciences et de Nanotechnologies, CNRS, Palaiseau, France)
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.23518
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.23518
 - **Abstract**
 Measurement-based quantum computing offers a promising route towards scalable, universal photonic quantum computation. This approach relies on the deterministic and efficient generation of photonic graph states in which many photons are mutually entangled with various topologies. Recently, deterministic sources of graph states have been demonstrated with quantum emitters in both the optical and microwave domains. In this work, we demonstrate deterministic and reconfigurable graph state generation with optical solid-state integrated quantum emitters. Specifically, we use a single semiconductor quantum dot in a cavity to generate caterpillar graph states, the most general type of graph state that can be produced with a single emitter. By using fast detuned optical pulses, we achieve full control over the spin state, enabling us to vary the entanglement topology at will. We perform quantum state tomography of two successive photons, measuring Bell state fidelities up to 0.80$\pm$0.04 and concurrences up to 0.69$\pm$0.09, while maintaining high photon indistinguishability. This simple optical scheme, compatible with commercially available quantum dot-based single photon sources, brings us a step closer to fault-tolerant quantum computing with spins and photons.
## Atom-light-correlated quantum interferometer with memory-induced phase comb
- **Keywords:** ['hybrid quantum']
 - **Authors:** Wenfeng Huang, Xinyun Liang, Jie Zhao, Zeliang Wu, Keye Zhang, Chun-Hua Yuan, Yuan Wu, Bixuan Fan, Weiping Zhang, Liqing Chen
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.23674
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.23674
 - **Abstract**
 Precise phase measurements by interferometers are crucial in science for detecting subtle changes, such as gravitational waves. However, phase sensitivity is typically limited by the standard quantum limit (SQL) with uncorrelated particles N. This limit can be surpassed using quantum correlations, but achieving high-quality correlations in large systems is challenging. Here, we propose and demonstrate an atom-light hybrid quantum interferometry whose sensitivity is enhanced beyond the SQL with atom-light quantum correlation and newly developed phase comb superposition via atomic-memory-assisted multiple quantum amplification. Finally, a phase sensitivity beyond the SQL of up to $8.3\pm 0.2$ dB is achieved, especially at $N=4 \times10^{13}/s$, resulting in both atomic and optical phase sensitivities of $6\times10^{-8} rad/\sqrt{Hz}$. This technique can advance sensitive quantum measurements in various fields.
## Robust quantum dots charge autotuning using neural network uncertainty
- **Keywords:** ['semiconductor spin qubit']
 - **Authors:** Victor Yon, Bastien Galaup, Claude Rohrbacher, Joffrey Rivard, Clément Godfrin, Ruoyu Li, Stefan Kubicek, Kristiaan De Greve, Louis Gaudreau, Eva Dupont-Ferrier, Yann Beilliard, Roger G. Melko, Dominique Drouin
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2406.05175
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2406.05175
 - **Abstract**
 This study presents a machine-learning-based procedure to automate the charge tuning of semiconductor spin qubits with minimal human intervention, addressing one of the significant challenges in scaling up quantum dot technologies. This method exploits artificial neural networks to identify noisy transition lines in stability diagrams, guiding a robust exploration strategy leveraging neural networks' uncertainty estimations. Tested across three distinct offline experimental datasets representing different single quantum dot technologies, the approach achieves over 99% tuning success rate in optimal cases, where more than 10% of the success is directly attributable to uncertainty exploitation. The challenging constraints of small training sets containing high diagram-to-diagram variability allowed us to evaluate the capabilities and limits of the proposed procedure.
